config/ext_jenkins-infra.yaml

```diff
@@ -225,7 +225,7 @@ controller:
                 sshKeysCredentialsId: "ec2-agents-privkey"
                 useInstanceProfileForCredentials: false
                 templates:
-                - ami: "ami-0f107be6f474c8c87" # https://github.com/jenkins-infra/packer-images/ LINUXAMD64 (DO NOT DELETE OR EDIT THIS COMMENT, USED BY UPDATECLI: https://github.com/jenkins-infra/kubernetes-management/blob/cd7e4132eef4095ca287e8c17b284d3c04234cb4/updatecli/updatecli.d/charts/jenkins-agents-infra-release.yaml#L73)
+                - ami: "{{ source "getLatestUbuntuAgentAMIAmd64" }}" # https://github.com/jenkins-infra/packer-images/ LINUXAMD64 (DO NOT DELETE OR EDIT THIS COMMENT, USED BY UPDATECLI: https://github.com/jenkins-infra/kubernetes-management/blob/cd7e4132eef4095ca287e8c17b284d3c04234cb4/updatecli/updatecli.d/charts/jenkins-agents-infra-release.yaml#L73)
```

Comment:
😢

Reply:
Strange... Cc @dduportal

Reply:
We already had this updatecli bug. @olblak had an explanation (something related to the `depends_on` not working as expected) but I can't remember the exact reason. 

When it happens here, it is because the packer-image tag did not finish to build so one of the AMI is not available yet. Maybe we could add a lock or something.